conf/services/conflateAdvOps.json

```diff
@@ -273,20 +273,20 @@
         "hoot_key": "ogr.split.o2s"
       },
       {
-        "id": "ogr_tds_add_fcsubtype",
+        "id": "ogr_esri_fcsubtype",
         "elem_type": "checkbox",
         "description": "Add the ESRI specific FCSUBTYPE field to the output",
-        "name": "OGR TDS Add FCSubtype",
+        "name": "OGR Esri FCSubtype",
```

Comment:
You need to update the cucumber test for this change:
>Unable to find css "label" with text "OGR TDS Add FCSubtype" (Capybara::ElementNotFound)
./features/step_definitions/custom_steps.rb:136:in `/^I should see checkbox "([^"]*)" (un)?checked$/'
features/advanced_options.feature:49:in `And I should see checkbox "OGR TDS Add FCSubtype" checked'